Makefile

```diff
@@ -276,6 +276,13 @@ api-fix-versions:
 api-doc:
 	./contrib/apiage.py --mode=write-doc
 
+api-report-updates: $(RESULTS_DIR)
+	./contrib/apiage.py --mode=find-updates \
+		--current-tag="$$(git describe --tags --abbrev=0)" \
+		> $(RESULTS_DIR)/updates-found.json
+	./contrib/apiage.py --mode=updates-to-markdown \
+		< $(RESULTS_DIR)/updates-found.json > $(RESULTS_DIR)/updates-found.md
+
 ifeq ($(RESULTS_DIR),)
 IMPLEMENTS_DIR:=$(PWD)/_results
 else
```

Comment:
We can even create a default `RESULTS_DIR` as __results_ in case not provided as input by moving the above block with an addition as follows:
```diff --git a/Makefile b/Makefile
index c165833..be185d3 100644
--- a/Makefile
+++ b/Makefile
@@ -200,6 +200,13 @@ test-containers-test: $(BUILDFILE) $(TEST_CTR_A) $(TEST_CTR_B)
                --ceph-conf=/ceph_a/ceph.conf \
                --mirror=/ceph_b/ceph.conf $(ENTRYPOINT_ARGS)
 
+ifeq ($(RESULTS_DIR),)
+IMPLEMENTS_DIR:=$(PWD)/_results
+RESULTS_DIR:=$(IMPLEMENTS_DIR)
+else
+IMPLEMENTS_DIR:=$(RESULTS_DIR)
+endif
+
 ifdef RESULTS_DIR
 $(RESULTS_DIR):
        mkdir -p $(RESULTS_DIR)

Reply:
Thanks, I'll think about it.

Reply:
Looking at how RESULTS_DIR is used elsewhere in the file, I'd rather not change it in this PR. Having to explicitly set RESULTS_DIR in the workflow yamls is small cost, auto-enabling RESULTS_DIR is going to side-effect all containers run from the makefile and I don't want to figure out all the implications that will have.

Setting IMPLEMENTS_DIR automatically was a compromise because we ask contributors to run `api-update`. In most other cases the `make api-*` commands are meant for automation or maintainers to run.

Similarly, I could just use "IMPLEMENTS_DIR" for all stuff related to the apiage commands even though api-report-updates isn't strictly using it for the implements.json file.  I'm leaning towards this middle ground now... but I'll give you some time to reply before I update the patches. If I don't hear back in a day or so I'll probably go with that option though.

Reply:
Let's move forward with the current approach and I'll follow up on my suggestion in a separate PR(after investigating possible implications if any).